package/boot/uboot-mediatek/patches/435-02-add-h3c_magic-nx30-pro-114m-ubootmod.patch

```diff
@@ -0,0 +1,425 @@
+--- /dev/null
```

Comment:
Please add symbol `CONFIG_SERIAL_RX_BUFFER=y`. ref: https://github.com/openwrt/openwrt/commit/3d8d807373746d6b3c1674bc450d78090954346f

Reply:
Now synced to latest main branch.